software-development-roles/development-operations/lead-devops.md

```diff
@@ -0,0 +1,183 @@
+    
+-   Build and maintain a diverse, inclusive culture within the local web operations community, growing awareness, inclusivity, and balance
+    
+-   Participate in support out of hours (for which you’ll be paid an allowance)
+    
+-   Coordinate and manage DevOps recruitment, shaping our in-house team, making it more diverse and inclusive
+
+## Person Specification  
+  
+
+### Essential:
+
+-   Development process optimisation - Aware of the importance consider developing process efficiency and the common ways in which processes are optimised. Supports specific activities to improve development processes. Able to spot or identify obvious deficiencies.
+    
+
+-   Modern standards approach - Understands key principles of modern standards approach and how they apply to the work they are undertaking, and will apply under guidance.
+
+-   Programming and build (software engineering) - Collaborates with others when necessary to review specifications and uses these agreed specifications to design, code, test and document programmes or scripts of medium to high complexity, using the right standards and tools.
+
+-   Service support - Identifies, locates and competently fixes faults.
+
+-   Systems design - Translates logical designs into physical designs. Produces detailed designs and documents all work using required standards, methods and tools, including prototyping tools where appropriate. Designs systems characterised by managed levels of risk, manageable business and technical complexity and meaningful impact. Works with well-understood technology and identifies appropriate patterns.
+
+-   Systems integration - Able to build and test simple interfaces between systems, or can work on more complex integration as part of a wider team.
+
+-   User focus - Identifies and engages with users or stakeholders to collate user needs evidence and understands and defines research which fits user needs. Able to use quantitative and qualitative data about users to turn user focus into outcomes.
+- Willingness to be assessed against [the requirements](https://www.gov.uk/government/publications/united-kingdom-security-vetting-clearance-levels/national-security-vetting-clearance-levels) for BPSS clearance
+
+### Desirable:
+
+-   Availability and capacity management - Manages the service components to ensure they meet business needs and performance targets.
+    
+-   Information security - Has an appreciation of information security, designing solutions and services with security controls embedded, specifically engineered as mitigation against security threats as core part of the solutions and services.
+
+-   Prototyping - Sees prototyping as a team activity, actively soliciting prototypes and testing with others. Establishes design patterns and iterates them. Knows a variety of methods of prototyping and chooses the most appropriate ones.
+
```

Comment:
Do we ever use desirable criteria? We might be able to remove these.

Reply:
Removed desirable criteria